CHANGELOG.MD

```diff
@@ -1,3 +1,9 @@
+## March 06, 2024
+- **Task**Multi-language - Create simple widget translation tables & API routes [DESENG-514](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-514)
+    - Added Widget translation model.
+    - Added Widget translation API.
+    - Added Unit tests.
+
 ## March 04, 2024
```

Comment:
Could I ask you to please combine the two Changelog entries for March 4th? I see that it was duplicated